lib/rules/web/admin_console/4.5/monitoring_alerts.xyaml

```diff
@@ -9,7 +9,31 @@ set_placeholder_input_value:
       xpath: (//input[@placeholder='<placeholder_text>'])[<index>]
     op: send_keys <input_value>
     type: input
-  
+check_alert_title:
+  elements:
+  - selector:
+      xpath: //*[contains(@class, 'pf-c-alert__title') and contains(text(), '<title>')]
+check_alert_description:
+  elements:
+  - selector:
+      xpath: //*[contains(@class, 'pf-c-alert__description')]//*[contains(., '<description>')]
+check_alert_message:
+  action:
+    if_param: title
+    ref: check_alert_title
+  action:
+    if_param: description
+    ref: check_alert_description
```

Comment:
These 3 rules I removed from your merged place: https://github.com/openshift/verification-tests/pull/1005/files#diff-77ea65010d594964ebd723929062da2dR96-R103
Is that okay for you?